app/Services/UserService.php

```diff
@@ -19,4 +29,38 @@ public function resource($id, $inputs = null)
 
         return $user;
     }
+
+    public function update($id, $inputs = null)
+    {
+        $user = Auth::user();
+
+        if (!empty($inputs['email']) && $inputs['email'] != $user->email) {
+            $this->userObj->whereId($user->id)->update(['email_verified_at' => null]);
+
+            $otp = mt_rand(100000, 999999);
```

Comment:
Let's create a helper file to generate the OTP. That method will expect a length of OTP.